plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/server/OpenShiftServerEditorSection.java

```diff
@@ -144,8 +146,18 @@ public void createSection(Composite parent) {
 		
 		section.setClient(container);
 
-		this.formPresenterSupport = new FormPresenterSupport(
-				new FormEditorPresenter(getManagedForm().getForm().getForm()), dbc);
+		IObservableValue<Boolean> dirtyStatusObservable = DataBindingUtils.createDirtyStatusObservable(getServerEditor());
+		dbc.addValidationStatusProvider(new MultiValidator() {
+
+			@Override
+			protected IStatus validate() {
+				if (Boolean.TRUE.equals(dirtyStatusObservable.getValue())) {
```

Comment:
can be just `if (dirtyStatusObservable.getValue()) {`

Reply:
I dont think so since dirtyStatus can also be just null.